### DISCUSSION TITLE
v3 request: ignore .bak files for discover

### DISCUSSION THREAD

**Discussion by wivaku** [filamentphp/filament#7340] [Help]
I sometimes quickly disable resources / widgets by renaming the file to .bak.
This worked fine in v2. In v3 it results in e.g. (after renaming to `Summary.php.bak`):

```
Class "App\Filament\Widgets\Summary.bak" does not exist

App\Providers\Filament\AdminPanelProvider:43
            ->discoverWidgets(in: app_path('Filament/Widgets'), for: 'App\\Filament\\Widgets')
```

Would be great if the `->discover...()` methods would only pick up files ending in `.php`
(and same for composer autoload)

**Comment by danharrin** [comment]
Feel free to PR

**Reply by wivaku** [reply]
done https://github.com/filamentphp/filament/pull/7770
